tor_ocr/main.py

```diff
@@ -76,8 +67,33 @@ def chunks(s, n):
         yield s[start:(start + n)]
 
 
-def run(config):
+def ocr_space_url(url, overlay=False, api_key=os.getenv("OCR_API_KEY", "helloworld"), language='eng'):
+    """ OCR.space API request with remote file.
```

Comment:
Almost. As in the example, please put the """ and "OCR.space API..." on two different lines with a blank line between "This code is stolen..." and ":param url:"